.github/workflows/_test_upstream_pax.yaml

```diff
@@ -30,12 +30,22 @@ on:
 
 jobs:
 
-  single-process-multi-device:
+  pax-single-process-multi-device:
     strategy:
       matrix:
-        PARALLEL_CONFIG:
-        - [1, 8, 1, 1]
-        - [1, 1, 2, 4]
+        include:
+          - TEST_NAME: 8DP1FSDP1TP1PP
```

Comment:
Do you change the test name?

Reply:
Yes, I have matrixified the args passed for this sub job as multi-node job and the JOB_NAME is defined as part of the matrix.